packages/components/src/components/DaCard.vue

```diff
@@ -207,14 +208,7 @@ export default {
   }
 }
 
-.card:hover,
-.card.hover {
-  & .card__background__image {
-    transform: translate(0, 4px) scale(1.05);
-  }
-}
-
-.animate-cards {
+.animate-cards .feed {
```

Comment:
Why did you add the `.feed` selector? I think it's too specific.
We can use cards potentially everywhere.

Reply:
Oh sorry, i was debugging something and wasn't able to remove after. Thanks for noticing!